src/components/search-form/search-form.html.twig

```diff
@@ -2,8 +2,9 @@
 
 {#
   Parameters:
-    - "text_input" (associative array) default: A predefined structure for EC Text Input
-    - "button" (associative array) default: A predefined structure for EC Button
+    - "label" (string) (default: '') Label of the search form
+    - "text_input" (associative array) (default: A predefined structure for EC Text Input)
```

Comment:
ECL maybe, why EC..? i know it was like this, but we have the chance to fix this.